lisa/microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -323,3 +330,16 @@ def _save_dmesg_logs(self, node: Node, log_path: Path) -> None:
         dmesg_path = log_path / "dmesg"
         with open(str(dmesg_path), "w", encoding="utf-8") as f:
             f.write(dmesg_str)
+
+    def _is_l1vh_partition(self, node: Node, log: Logger) -> bool:
+        """Check if the node is L1VH parent partition."""
+        l1vh_pattern = re.compile(
```

Comment:
It looks just a string match, `if "Hyper-V: running as L1VH partition" in dmesg_log` should be enough.

Reply:
Added an example.